src/Components/Patient/FileUpload.tsx

```diff
@@ -1115,183 +1121,184 @@ export const FileUpload = (props: FileUploadProps) => {
-                  ></path>
-                </svg>
-                Proceed
-              </button>
-              <button
-                type="button"
-                className="btn-danger btn mr-2 w-full md:w-auto"
-                onClick={(_) => setModalOpenForArchive(false)}
-              >
-                Cancel
-              </button>
+            <div className="text-sm text-grey-200">
+              <h1 className="text-xl text-black">File Details</h1>
+              This file is archived. Once a file is archived it cannot be
+              unarchived.
             </div>
-          </form>
-        </div>
-      </Modal>
-      <Modal open={modalOpenForMoreDetails}>
-        <div className="h-screen w-full absolute flex items-center justify-center">
-          <div className="bg-white rounded shadow p-8 m-4 max-h-full flex flex-col max-w-lg w-2/3 min-w-max-content">
-            <div>
-              <div className="text-center">
-                <i className="fa-solid fa-file-circle-xmark my-2 fa-4x text-primary-500"></i>
-              </div>
-              <div className="text-md text-center m-2">
-                <b>{modalDetails?.name} file is archived.</b>
-              </div>
-              <div className="text-md text-center">
-                <b>Reason:</b> {modalDetails?.reason}
-              </div>
-              <div className="text-md text-center">
-                <b>Archived_by:</b> {modalDetails?.userArchived}
-              </div>
-              <div className="text-md text-center">
-                <b>Time of Archive:</b>
-                {formatDate(modalDetails?.archiveTime)}
-              </div>
+          </div>
+        }
+        onClose={() => setModalOpenForMoreDetails(false)}
+      >
+        <div className="flex flex-col">
+          <div>
+            <div className="text-md text-center m-2">
+              <b>{modalDetails?.name}</b> file is archived.
+            </div>
+            <div className="text-md text-center">
+              <b>Reason:</b> {modalDetails?.reason}
+            </div>
+            <div className="text-md text-center">
+              <b>Archived_by:</b> {modalDetails?.userArchived}
             </div>
-            <div className="flex flex-col-reverse md:flex-row gap-2 mt-4 justify-end">
-              <button
-                type="button"
-                className="btn-danger btn mr-2 w-full md:w-auto"
-                onClick={(_) => setModalOpenForMoreDetails(false)}
-              >
-                Close
-              </button>
+            <div className="text-md text-center">
+              <b>Time of Archive:</b>
+              {formatDate(modalDetails?.archiveTime)}
             </div>
           </div>
+          <div className="flex flex-col-reverse md:flex-row gap-2 mt-4 justify-end">
+            <ButtonV2
+              type="button"
+              variant="danger"
```

Comment:
Close button is not under danger context. Use secondary 
```suggestion
              variant="secondary"
```